.github/workflows/core_internal.yml

```diff
@@ -0,0 +1,79 @@
+name: core_internal
```

Comment:
Don't we need a `core_extension.yml` also?

Reply:
Yes, added in 0826e4b42c1d929a4b926180287d428ecb8f7a58.

Currently, the workflow is set to lint the `FirebaseCoreExtension.podspec` (which currently only contains source headers). IIUC, I can't just build the source for SPM (no scheme generates for the `FirebaseCoreExtension` target). I could think of three options:
1. No op– don't run `spm` workflows for FirebaseCoreExtension.
2. Add a `FirebaseCoreExtensionTesting` scheme that has no tests but builds `FirebaseCoreExtension`.
3. Add a `FirebaseCoreExtensionTesting` scheme and rearrange the `FirebaseCore/Extension` directory and add API build tests for both Swift and ObjC? The package modifications would look something like:
<img width="856" alt="image" src="https://user-images.githubusercontent.com/36927374/159474838-0474daf4-a40a-4bf7-940d-35a87a3adb50.png">

WDYT?

Reply:
Since it's only headers, `pod lib lint` testing should be sufficient.

Reply:
Done in [e9483fa](https://github.com/firebase/firebase-ios-sdk/pull/9482/commits/e9483fa2565916206cf61e0763ac52f9354e5db1)